docs/tools/qemu.md

```diff
@@ -41,29 +46,37 @@ For instance, the following command will run QEMU with
 	-device scsi-block,bus=scsi0.0,drive=zbc0
 ```
 
-The first line `-device pcie-root-port,bus=pcie.0,id=rp1` creates a PCIe root
-complex. The second line `-device virtio-scsi-pci,bus=pcie.0,id=scsi0` defines
-a virtio adapter connected to the PCIe bus previously defined. Finally, the last
-2 lines `-drive file=/dev/sdf,format=raw,if=none,id=zbc0` and
-`-device scsi-block,bus=scsi0.0,drive=zbc0` define the device connected to the
-virtio SCSI adapter. In this example, since `-device scsi-block` is used, the
-host device to attach is specified using the device block device file
-(`/dev/sdf`) in this example.
+#### Analysis of QEMU Command
+
+The first line, `-device pcie-root-port,bus=pcie.0,id=rp1`, creates a PCIe root
+complex. 
+
+The second line, `-device virtio-scsi-pci,bus=pcie.0,id=scsi0`,
+defines a virtio adapter that connects to the previously-defined PCIe bus.
+
+The last 2 lines, `-drive file=/dev/sdf,format=raw,if=none,id=zbc0`
+and `-device scsi-block,bus=scsi0.0,drive=zbc0`, define the device that is
+connected to the virtio SCSI adapter. In this example, `-device scsi-block` is
+used, which means that the host device that will be attached is specified with 
+the device block device file (`/dev/sdf`).
 
 See <a href="https://github.com/qemu/qemu/blob/master/docs/pcie_pci_bridge.txt"
 target="_blank">here</a> for a detailed description of these options.
 
-Though this method allows attaching a zoned block disk to a QEMU guest, SCSI
-command sense data is not processed correctly in QEMU versions prior to version
-4.1. This causes the guest operating system to hang if the guest attempts to
-access a command sense data (for instance upon command failures). This
-attachment method should thus be avoided with versions of QEMU older than
-version 4.1.
+### Sense data in QEMU prior to Version 4.1
+
+Although this method makes it possible to attach a zoned block disk to a QEMU
+guest, SCSI command sense data is not processed correctly in older QEMU
+versions (this includes all QEMU versions prior to version 4.1). This causes
+the guest operating system to hang if the guest attempts to access command
+sense data (for example, after command failures). Because of older QEMU
+versions' inability properly to process SCSI command sense data, you should
```

Comment:
s/properly to/to properly

Reply:
I was probably being too pedantic avoiding this split infinitive. I'll put it back the other way.